ui/src/app/views/workspace-edit/component.ts

```diff
@@ -242,6 +243,11 @@ export class WorkspaceEditComponent implements OnInit {
     if (this.mode === WorkspaceEditMode.Edit) {
       this.workspace = wsData;
       this.accessLevel = wsData.accessLevel;
+      if (!this.workspace.description && this.workspace.description === '') {
```

Comment:
When will you ever have both of these? Could you just check for `this.workspace.description === ''`?